R/validate_per.R

```diff
@@ -6,7 +6,8 @@
 #' @param missing_per Is `per`` missing in the outer function? If so the defualt for that function will be used.
 #' @param .fortnights_per_yr What is the ratio of the fortnightly payment amount to the yearly payment amount? By default, 26. (Some payments expect 26; others expect 364/14.)
 #' 
-#' @author Matthew Katzen
+#' @examples 
+#' See rent_assistance function
```

Comment:
Entries under `@examples` are run as R code, so this will error. Instead put

```r
#' @details For examples, see \code{\link{rent_assistance}}.
```